src/seqme/models/hyformer.py

```diff
@@ -0,0 +1,191 @@
+from enum import Enum
+
+import numpy as np
+import torch
+from tqdm import tqdm
+
+from .exceptions import OptionalDependencyError
+
+
+class HyformerCheckpoint(str, Enum):
+    # molecules checkpoints
+    molecules_8M = "SzczurekLab/hyformer_molecules_8M"
+    molecules_50M = "SzczurekLab/hyformer_molecules_50M"
+
+    # peptides checkpoints
+    peptides = "SzczurekLab/hyformer_peptides"
+    peptides_mic = "SzczurekLab/hyformer_peptides_mic"
```

Comment:
Can you add additional descriptive text about the checkpoints e.g., embedding dimension, data trained on (if relevant) and for the "hyformer_peptides_mic" checkpoint what strain(s) it predicts the MIC for?

Reply:
done